commit 4d3d6b0c9eb6f0117b3de72f2db5b6af4ccd4f3b (refs/changes/98/4798/4)
Author: dyep <dyep49@gmail.com>
Date:   2018-10-18T14:48:16-07:00 (12 months ago)
    
    OS-7220 Snapshots are not unmounted on VM pre-halt
    Reviewed by: Rob Johnston <rob.johnston@joyent.com>

diff --git a/overlay/generic/usr/lib/brand/jcommon/statechange b/overlay/generic/usr/lib/brand/jcommon/statechange
index da3bd811..58eae5ab 100644
--- a/overlay/generic/usr/lib/brand/jcommon/statechange
+++ b/overlay/generic/usr/lib/brand/jcommon/statechange
@@ -876,7 +876,7 @@ pre)
 		setup_fs
 		;;
 	4)	# pre-halt
-		[[ -n "$jst_snowsnap" ]] && cleanup_snapshots
+		[[ -n "$jst_showsnap" ]] && cleanup_snapshots
 		cleanup_net
 		;;
 	esac
